rfcs/53-generalized-hierarchy.md

```diff
@@ -0,0 +1,273 @@
+# Feature Name: `generalized-hierarchies`
+
+## Summary
+Create a general hierarchy or hierarchies for Bevy worlds that is ergonomic to
```

Comment:
This really only covers the "objective". Good summaries should at least hint at how we got there.

Reply:
done!